pkgs/desktops/deepin/default.nix

```
@@ -1,9 +1,20 @@
 { lib, pkgs, libsForQt5 }:
 let
+  functions = {
+    replaceAll = x: y: ''
+      echo Replacing "${x}" to "${y}":
+      for file in $(grep -rl "${x}")
+      do
+        echo -- $file
+        substituteInPlace $file \
+          --replace "${x}" "${y}"
+      done
```

```suggestion
      for file in $(grep -rl "${x}"); do
        echo -- $file
        substituteInPlace $file \
          --replace "${x}" "${y}"
      done
```
we should consider using ripgrep to speed things up by a lot if there are many files.

---

ripgrep not in stdenv, I think grep is enough